app/models/participant_declaration.rb

```diff
@@ -8,9 +8,15 @@ class ParticipantDeclaration < ApplicationRecord
   scope :active_for_lead_provider, ->(lead_provider) { active.for_lead_provider(lead_provider) }
   scope :count_active_for_lead_provider, ->(lead_provider) { active_for_lead_provider(lead_provider).count }
   scope :count_active_for_lead_provider_between, ->(lead_provider, start_date, end_date) { declared_as_between(start_date, end_date).count_active_for_lead_provider(lead_provider) }
+  scope :count_active_uplift_for_lead_provider, ->(lead_provider) { active_uplift_for_lead_provider(lead_provider).count }
```

Comment:
Suggestion to call `.count` from calling code instead of having another scope here